daml-assistant/integration-tests/src/Main.hs

```diff
@@ -74,11 +74,13 @@ tests damlDir tmpDir = testGroup "Integration tests"
     , packagingTests tmpDir
     , quickstartTests quickstartDir mvnDir
     , cleanTests cleanDir
+    , deployTest deployDir
     ]
     where quickstartDir = tmpDir </> "q-u-i-c-k-s-t-a-r-t"
```

Comment:
waa!! how did this end up like this?!

Reply:
@nickchapman-da changed that to test that we do not depend on the directory name in the quickstart-java name.

Reply:
yes, it was me!

Reply:
Cool :-) that was unexpected!